jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-backend/src/main/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImpl.java

```diff
@@ -96,8 +98,9 @@ public CaseDefinitionSummary getCaseDefinition(final String serverTemplateId, fi
     }
 
     @Override
-    public String startCaseInstance(final String serverTemplateId, final String containerId, final String caseDefinitionId) {
-        return client.startCase(containerId, caseDefinitionId);
+    public String startCaseInstance(final String serverTemplateId, final String containerId, final String caseDefinitionId, final String owner) {
+        final CaseFile caseFile = CaseFile.builder().addUserAssignments(CASE_ROLE_OWNER, owner).build();
```

Comment:
Variables names were switched in the API, fixed in here: https://github.com/droolsjbpm/droolsjbpm-integration/pull/810